tests/standalone/Boto3Using.py

```diff
@@ -0,0 +1,103 @@
+import boto3
+from moto import mock_s3
+
+# nuitka-skip-unless-imports: boto3,moto
+
+def getClient():
+    """
+    returns the boto3 client
+    """
+    return boto3.client('s3')
+
+def listS3Buckets():
+    """
+    lists s3 bucket names
+    """
+    s3 = getClient()
+
+    response = s3.list_buckets()
+    if response:
+        for bucket in response.get('Buckets', []):
+            yield bucket['Name']
+
+def listS3Objects(bucket):
+    """
+    lists s3 objects in a bucket
+    """
+    s3 = getClient()
+
+    response = s3.list_objects(Bucket=bucket)
+    if response:
+        for _object in response.get('Contents', []):
+            yield _object['Key']
+
+def readS3Object(bucket,key):
+    """
+    returns the content of a s3 file
+    """
+    s3 = getClient()
+
+    response = s3.get_object(Bucket=bucket, Key=key)
+    if response:
+        return response['Body'].read()
+
+
+
+if __name__ == '__main__':
```

Comment:
That condition is not very useful to have, can just put things on the module level.